client/src/pages/HomePage/HomePage.js

```diff
@@ -1,10 +1,23 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import axios from "axios";
 import HomeHeader from './sections/HomeHeader/HomeHeader';
 import MealSearchForm from './sections/MealSearchForm/MealSearchForm'
 import {Footer} from '../../components'
 import {FooterWrapper} from '../../components/Footer/FooterWrapper'
 import {StyledPara} from './HomePage.style'
+import findMeals from '../../actions/findMeals'
 
+const HomePage = (props) => {
+  const submit = (values) => {
+    const {postcode,deliveryDate} = values
+    axios(`http://api.postcodes.io/postcodes/${postcode}`)
+    .then(({data:{result:location}})=> {
+      props.findMeals({location, deliveryDate})
+    })
+    .catch(console.log)
```

Comment:
You need to display a message to the user that there was an error, could open an issue about it as a stretch goal

Reply:
Done here: #73